doc/en/user/source/extensions/importer/rest_examples.rst

```diff
@@ -566,4 +566,60 @@ a lot of shapefiles to process, also the import run will be done in asynchronous
     curl -u admin:geoserver -XPOST "http://localhost:8080/geoserver/rest/imports/0?async=true"
     
 The response will return immediately in this case as well, and the progress can be followed as the
-tasks in the import switch state.
\ No newline at end of file
+tasks in the import switch state.
+
+
+Importing and optimizing a large image with a single request
+------------------------------------------------------------
+
+A large image appears every now and then on a mounted disk share, the image needs to be
+optimized and imported into GeoServer as a new layer.
+Since the source is large and we need to copy it on the local disk where the data dir resides,
+a "remote" data is the right tool for the job, an asynchronous execution is also recommended
+to avoid waiting on a possibly large command.
+In this case the request will also contains the "exec=true" parameter to force the importer
+an immediate execution of the command.
+
+The request will then look as follows::
+
+    curl -u admin:geoserver -XPOST -H "Content-type: application/json" -d @import.json "http://localhost:8080/geoserver/rest/imports?async=true&exec=true"
```

Comment:
I've modified the code instead, since the other parameters are already in abbreviated form (async instead of asynchronous)